semanticdb/metacp/src/main/scala/scala/meta/internal/javacp/Javacp.scala

```diff
@@ -350,7 +360,7 @@ object Javacp {
     asmName
   }
 
-  private def ssym(asmName: String): String = {
+  private def ssym(asmName: String, scope: Scope): String = {
```

Comment:
How about turning Javacp from an object to a class like Scalacp. This way we won't have to pass `scope` to `ssym`, and the `Scope` itself won't have to be stretched to also accommodate inner classes (which turns it from a `Scope` to more like a `SymtabLite`).